src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,6 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isMutating() { return false; }
```

Comment:
Would making this command abstract be more appropriate?
By making this command abstract, you can force child classes to declare if they're mutating, instead of relying on the parent class.